### PR TITLE
Test Case for Http 201 Response for Relative URL location Header 

### PR DESCRIPTION
**Pull request by ChamNDeSilva**
Related to the fix of public JIRA [1].

[1] https://wso2.org/jira/browse/APIMANAGER-4023

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/synapseconfigs/rest/dummy_api_relative_url_loc_header.xml [hunk 1]
```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (harsha89) [inline modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/synapseconfigs/rest/dummy_api_relative_url_loc_header.xml]: it it require to include licence text here?

Comment (ChamNDeSilva) [inline modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/synapseconfigs/rest/dummy_api_relative_url_loc_header.xml]: Fixed in PR https://github.com/wso2/product-apim/pull/395

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/RelativeUrlLocationHeaderTestCase.java [hunk 2]
```diff
@@ -0,0 +1,185 @@
+/*
+ *   Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (prabathabey) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/RelativeUrlLocationHeaderTestCase.java]: License is not properly formatted.

Comment (ChamNDeSilva) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/RelativeUrlLocationHeaderTestCase.java]: Fixed in https://github.com/wso2/product-apim/pull/403

**Pull request merged**